### PR TITLE
Minor improvements to Spanish translation

### PR DESCRIPTION
**Pull request by Perboire99**
Updated some translations to correct some minor mistakes.

### CODE REVIEW

File: web/locales/es.json [hunk 1]
```diff
@@ -1452,283 +1453,283 @@
...
             "zoneName": "City Of Tacoma, Department Of Public Utilities, Light Division"
         },
         "US-NW-WACM": {
-            "countryName": "United States of America",
+            "countryName": "Estados Unidos",
             "zoneName": "Western Area Power Administration - Rocky Mountain Region"
         },
         "US-NW-WAUW": {
-            "countryName": "United States of America",
+            "countryName": "Estados Unidos",,
```
Comment (veqtrus) [inline web/locales/es.json:1664]: An extra comma slipped in here

Comment (Perboire99) [reply web/locales/es.json]: Thanks! It's now solved :)

**Review by FelixDQ — APPROVED** [review]
LGTM!

**Comment by veqtrus** [comment]
> Autosquash / Autosquash (pull_request) Failing after 3s — Autosquash

Are you sure that pressing the `Update branch` button isn't quicker?

**Comment by FelixDQ** [comment]
> > Autosquash / Autosquash (pull_request) Failing after 3s — Autosquash
> 
> Are you sure that pressing the `Update branch` button isn't quicker?

Yea, it doesn't seem to work on public repos for some reason. We just use this internally, so that we don't have to
1) Update branch
2) Wait 5 minutes for test to complete
3) Merge
4) Repeat for next pr.

I'm removing it again in https://github.com/tmrowco/electricitymap-contrib/pull/2736

**Pull request merged**